### DISCUSSION TITLE
Roles in game design

### DISCUSSION THREAD

**Discussion by jeffaroundtheworld** [jeffaroundtheworld/2024gamecollab#7] [General]
With a large group of people working together, we are going to need to break down the giant task of building a game into different roles.  While everyone is welcome to and can contribute in different areas... it would be handy to know what you are particularly interested in.

Here are the general roles (that are applicable to our group - no need for a marketing manager here) in a game development team:

**- Game Designer**: Responsible for conceptualizing game ideas, creating game mechanics, and designing gameplay elements.
**- Game Developer/Programmer**: Writes the code that brings the game to life, implementing features, mechanics, and systems.
**- Artist/Graphic Designer**: Creates visual elements such as characters, environments, animations, user interfaces, and promotional materials.
**- Animator**: Specializes in creating character animations, object animations, and visual effects within the game.
- **Sound Designer/Audio Engineer:** Designs and implements sound effects, music, and other auditory elements to enhance the gaming experience.
- **Writer/Story Designer**: Develops the narrative, dialogues, and lore for the game, ensuring cohesion with the gameplay and overall vision.
- **Level Designer**: Constructs the game levels, including layout, objectives, challenges, and pacing, to provide an engaging player experience.
- **Quality Assurance (QA) Tester**: Tests the game thoroughly to identify bugs, glitches, and areas for improvement, ensuring a polished final product.
- **Producer/Project Manager**: Oversees the development process, manages timelines, budgets, and resources, and facilitates communication between team members.

Reply to this discussion with the roles that most interest you (and any roles that you are definitely NOT interested in).

**Comment by kodriq** [comment]
writing/story design and art is something i could do a lot of
does anyone else draw/write?

**Reply by ediehouston** [reply]
I can for sure help with writing the story if you need it at all

**Reply by mya1919** [reply]
me too

**Reply by kodriq** [reply]
would u guys wanna have some sort of gorupchat to discuss the plot on like gmail or smth

**Reply by ediehouston** [reply]
yeah I think that would be a really good idea

**Reply by ediehouston** [reply]
what should we have the gc on

**Reply by kodriq** [reply]
i can make one in like gmail chats or we could use insta or smth

**Reply by mya1919** [reply]
either works for me

**Reply by kodriq** [reply]
i think we should do insta if u guys have it

**Reply by mya1919** [reply]
sounds good
mine is mya_j.s

**Reply by kodriq** [reply]
mine is kidneyrails

**Reply by ediehouston** [reply]
yeah I feel like I would check insta more, mine is edieh0uston

**Reply by mya1919** [reply]
ok ill make it

**Reply by JackV13** [reply]
Mine is jackvallance13

**Reply by zorienm** [reply]
mine is zori.en

**Reply by ajleach9689** [reply]
mines ajleach_

**Reply by Rhyswatsonn** [reply]
rhys.watson_

**Reply by kodriq** [reply]
ill add u guys

**Reply by ZachLife** [reply]
zach.stevenson_

**Reply by Jackcarlisle1** [reply]
jackcarlisle3

**Comment by Jackcarlisle1** [comment]
I have access to my dads fancy ass modeling software that i have recently started learning how to use. So I could make like the map and stuff and import it in. it might take a bit for me to make it because I basically have no experience but if you guys are fine with making a small map just for the early development then just adding the map and making it work later id be happy to put more of my effort into that.

**Reply by kodriq** [reply]
that would be super cool
what software is it?

**Reply by Jackcarlisle1** [reply]
houdini

**Comment by ediehouston** [comment]
I can do sound and audio and help with the writing of the story if needed at all

**Comment by mya1919** [comment]
I could do some of the graphic design for the characters or environment

**Comment by Rhyswatsonn** [comment]
my strong suit is probably level design and audio but I am okay to do anything needed

**Reply by ZachLife** [reply]
Same with me

**Comment by zorienm** [comment]
I could try coding the player

**Reply by ajleach9689** [reply]
I can help with that too

**Comment by JackV13** [comment]
I could be the Quality Assurance Tester

**Reply by JackV13** [reply]
or maybe the 2D enemy

**Comment by JackV13** [comment]
i could make the flashlight mechanic.
